frontend/src/components/layout/AppNavBar/AppNavBar.tsx

```diff
@@ -134,12 +134,19 @@ function AdminDropdown() {
       >
         Administrative Areas
       </NavDropdown.Item>
-      <NavDropdown.Item
-        data-testid="navbar-admin-upload-properties"
-        onClick={() => navigate('/admin/uploadProperties')}
-      >
-        Upload Properties
-      </NavDropdown.Item>
+      {
+        // Only allow System Administrators to see this item
+        keycloak.hasRole('System Administrator') ? (
```

Comment:
Is this not the same as `keycloak.isAdmin` ?

Reply:
Not quite. `isAdmin` also allows another role.

<img width="402" alt="image" src="https://github.com/bcgov/PIMS/assets/37922247/f8e0a680-6c3a-43cf-a78b-9ae75214e363">